statistics/update.go

```diff
@@ -193,8 +193,11 @@ func (s *SessionStatsCollector) StoreQueryFeedback(feedback interface{}, h *Hand
 	} else {
 		rate = math.Abs(expected-float64(q.actual)) / float64(q.actual)
 	}
-	if rate >= MinLogErrorRate && (q.actual >= MinLogScanCount || q.expected >= MinLogScanCount) && log.GetLevel() == log.DebugLevel {
-		q.logDetailedInfo(h)
+	if rate >= MinLogErrorRate && (q.actual >= MinLogScanCount || q.expected >= MinLogScanCount) {
+		metrics.HighErrorRateFeedBackCounter.Inc()
+		if log.GetLevel() == log.DebugLevel {
```

Comment:
Can we change this log level to INFO or WARN?

Reply:
This part logs many things and has some heavy calculation. So it would be better to set it to debug level.